### PR TITLE
Use stable rust

### PR DESCRIPTION
**Pull request by rhdxmr**
@rsdy 

- As discussed in #6, change fixed rust version to current stable release.
- And Ubuntu 20.04 LTS, 18.04 LTS are added.

### CODE REVIEW

**Review by rsdy — APPROVED** [review]

**Comment by rhdxmr** [comment]
@rsdy 
It is good to see this PR is approved. But I don't have write access to this repo so I can't merge this.
Could you merge this?

**Pull request merged**
